docs/troubleshooting.md

```diff
@@ -408,6 +408,8 @@ Running Puppeteer on Heroku requires some additional dependencies that aren't in
 
 The url for the buildpack is https://github.com/jontewks/puppeteer-heroku-buildpack
 
+Ensure that you're using `'--no-sandbox'` mode when launching pupeteer. You can achieve it by passing it as an argument to your `.launch()` call, like this `puppeteer.launch({args: ['--no-sandbox']});`
```

Comment:
```suggestion
Ensure that you're using `'--no-sandbox'` mode when launching Puppeteer. This can be done by passing it as an argument to your `.launch()` call: `puppeteer.launch({ args: ['--no-sandbox'] });`.
```

Reply:
Thank you for your suggestions! And sorry for the typos.